packages/shared/src/components/cards/common.tsx

```diff
@@ -69,3 +69,12 @@ export const generateTitleClamp = ({
 
   return hasHtmlContent ? 'line-clamp-4' : 'line-clamp-9';
 };
+
+export enum FeedItemType {
+  RegularPost = 'post',
+  RegularAd = 'ad',
```

Comment:
do we need Regular on the naming? I think it's not necessary since other items have other names

Reply:
Same feelings :)

Reply:
It just interferes with the imported `Post` type, I should maybe find another file to put this enum with where the `Post` is not imported, but this is already the `common.tsx`. I thought of adding `types.ts` file but this is not even just a type.

Reply:
I can disable the link here since this is something that doesn't affect negatively. Thoughts?
![Screenshot 2024-05-22 at 6 00 48 PM](https://github.com/dailydotdev/apps/assets/13744167/ac117317-2764-40e4-b9e6-ac7d138bbed7)

Reply:
Maybe if u save it as a string 'Post', no need to disable eslint